### PR TITLE
CLN: Finish isort core

### PR DESCRIPTION
**Pull request by alimcmaster1**
- [x] xref #23334
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: pandas/core/indexes/frozen.py [hunk 1]
```diff
@@ -9,10 +9,15 @@
 """
 
 import warnings
+
 import numpy as np
-from pandas.core.base import PandasObject
+
 from pandas.util._decorators import deprecate_kwarg
+
```
Comment (WillAyd) [inline pandas/core/indexes/frozen.py]: Hmm what's with the empty lines between `from` statements here? Not seeing that anywhere else?

File: pandas/core/indexes/numeric.py [hunk 2]
```diff
@@ -1,26 +1,22 @@
 import warnings
 
 import numpy as np
+
 from pandas._libs import index as libindex
+from pandas.util._decorators import Appender, cache_readonly
```
Comment (WillAyd) [inline pandas/core/indexes/numeric.py]: Isn't this supposed to be alphabetical? Should come after the following import, no?

File: pandas/core/indexes/frozen.py (hunk 1 above)
Comment (alimcmaster1) [reply pandas/core/indexes/frozen.py]: These are separate sections see the isort section in the setup.cfg for latest spec.

**Comment by alimcmaster1** [comment]
Reopen

**Comment by alimcmaster1** [comment]
Thanks for taking a look @WillAyd just actually removed a bunch of files to aid reviewing this

**Comment by jreback** [comment]
thanks @alimcmaster1 

btw these are going really well, don't mind having more files in each PR.

**Comment by jreback** [comment]
ping on green.

**Comment by jreback** [comment]
small linting issue in the 3.6 build

run

``LINT=1 ci/code_checks.sh``

**Comment by jreback** [comment]
thanks @alimcmaster1

**Pull request merged**